features/built_in_matchers/include.feature

```diff
@@ -142,3 +142,58 @@ Feature: `include` matcher
       | expected {:a => 7, :b => 5} to include {:d => 3}              |
       | expected {:a => 7, :b => 5} not to include {:a => 7}          |
 
+  Scenario: pseudo-hash usage
+    Given a file named "pseudo_hash_include_matcher_spec.rb" with:
+      """ruby
+      require 'delegate'
+      class PseudoHash < SimpleDelegator
+      end
+
+      RSpec.describe PseudoHash.new(:a => 7, :b => 5) do
+        it { is_expected.to include(:a) }
+        it { is_expected.to include(:b, :a) }
+        it { is_expected.to include(:a => 7) }
+        it { is_expected.to include(:b => 5, :a => 7) }
+        it { is_expected.not_to include(:c) }
+        it { is_expected.not_to include(:c, :d) }
+        it { is_expected.not_to include(:d => 2) }
+        it { is_expected.not_to include(:a => 5) }
+        it { is_expected.not_to include(:b => 7, :a => 5) }
+
+        # deliberate failures
+        it { is_expected.not_to include(:a) }
+        it { is_expected.not_to include(:b, :a) }
+        it { is_expected.not_to include(:a => 7) }
+        it { is_expected.not_to include(:a => 7, :b => 5) }
+        it { is_expected.to include(:c) }
+        it { is_expected.to include(:c, :d) }
+        it { is_expected.to include(:d => 2) }
+        it { is_expected.to include(:a => 5) }
+        it { is_expected.to include(:a => 5, :b => 7) }
+
+        # Mixed cases--the hash includes one but not the other.
+        # All 4 of these cases should fail.
+        it { is_expected.to include(:a, :d) }
+        it { is_expected.not_to include(:a, :d) }
+        it { is_expected.to include(:a => 7, :d => 3) }
+        it { is_expected.not_to include(:a => 7, :d => 3) }
+      end
+      """
+    When I run `rspec pseudo_hash_include_matcher_spec.rb`
+    Then the output should contain all of these:
+      | 22 examples, 13 failures                                      |
+      | expected {:a => 7, :b => 5} not to include :a                 |
+      | expected {:a => 7, :b => 5} not to include :b and :a          |
+      | expected {:a => 7, :b => 5} not to include {:a => 7}          |
+      | expected {:a => 7, :b => 5} not to include {:a => 7, :b => 5} |
+      | expected {:a => 7, :b => 5} to include :c                     |
+      | expected {:a => 7, :b => 5} to include :c and :d              |
+      | expected {:a => 7, :b => 5} to include {:d => 2}              |
+      | expected {:a => 7, :b => 5} to include {:a => 5}              |
+      | expected {:a => 7, :b => 5} to include {:a => 5, :b => 7}     |
+      | expected {:a => 7, :b => 5} to include :d                     |
+      | expected {:a => 7, :b => 5} not to include :a                 |
+      | expected {:a => 7, :b => 5} to include {:d => 3}              |
+      | expected {:a => 7, :b => 5} not to include {:a => 7}          |
+
+
```

Comment:
IMO, this isn't worth elevating to such a prominent place in our relish docs.  We don't show every last use case of the matchers in our relish docs, and there's a definite maintenance cost to adding to our cucumber suite.  Do you mind removing this scenario?

Reply:
Agreed. Removed now.